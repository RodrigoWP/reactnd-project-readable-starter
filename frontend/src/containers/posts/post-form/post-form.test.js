'use strict'

import React from 'react'
import { shallow } from 'enzyme'
import PostForm from '.'

it('Should render correctly', () => {
  const component = (
    <PostForm
      onClickClose={() => jest.fn()}
      handleRefresh={() => jest.fn()}
    />
  )

  expect(shallow(component)).toMatchSnapshot()
})
