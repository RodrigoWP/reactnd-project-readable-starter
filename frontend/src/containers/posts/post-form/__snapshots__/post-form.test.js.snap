// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render correctly 1`] = `
<Modal
  onClickClose={[Function]}
  open={true}
  title="Novo post"
>
  <form
    onSubmit={[Function]}
  >
    <Row>
      <Col
        xs={true}
      >
        <Input
          label="title"
          name="title"
          onChange={[Function]}
          required={true}
          value=""
        />
      </Col>
      <Col
        xs={true}
      >
        <Input
          label="author"
          name="author"
          onChange={[Function]}
          required={true}
          value=""
        />
      </Col>
    </Row>
    <Row>
      <Col
        xs={true}
      >
        <SelectBox
          label="Category"
          onChange={[Function]}
          options={
            Array [
              Object {
                "label": "react",
                "value": "react",
              },
              Object {
                "label": "redux",
                "value": "redux",
              },
              Object {
                "label": "udacity",
                "value": "udacity",
              },
            ]
          }
          placeholder="Select a category"
          required={true}
          value="react"
        />
      </Col>
    </Row>
    <Row>
      <Col
        xs={true}
      >
        <Textarea
          label="posting"
          name="body"
          onChange={[Function]}
          required={true}
          value=""
        />
      </Col>
    </Row>
    <div>
      <Button
        htmlType="submit"
        label="Salvar"
        primary={true}
      />
    </div>
  </form>
</Modal>
`;
